### PR TITLE
Support more operators

### LINKED ISSUE #16: Add unary plus and minus
- Reference implementation: algobot76/chibicc@c6bd5cd7a92846e4d38d187167792ff83f0d5b92
- Blocked by #7

### PR DESCRIPTION
**Pull request by algobot76**
Close #16

### CODE REVIEW

File: pylintrc [hunk 1]
```diff
@@ -271,7 +273,7 @@ generated-members=
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=80
+max-line-length=88
```
Comment (omar2535) [inline pylintrc:276]: Can we just do 90?

**Review by omar2535 — CHANGES_REQUESTED** [review]

File: pylintrc (hunk 1 above)
Comment (algobot76) [reply pylintrc:276]: See: https://black.readthedocs.io/en/stable/compatible_configs.html#pylint

Comment (algobot76) [reply pylintrc:276]: we could use `--line-length` of `black` to achieve a maximum line length of 90 though...

Comment (algobot76) [reply pylintrc:276]: More details on this issue: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length

Comment (omar2535) [reply pylintrc:276]: Hm let's just keep `88`

**Review by omar2535 — APPROVED** [review]

**Pull request merged**